### DISCUSSION TITLE
TOGA reference guide

### DISCUSSION THREAD

**Discussion by Gergely54** [beeware/toga#1781] [Q&A]
Hi, I'm new on TOGA.
So, I'm looking for a TOGA reference guide. The TOGA Readthedocs are not sufficient for me...
Please advice where I can find more info.
Thanks, Gergely
gergely@vegh.nl

**Comment by freakboy3742** [comment]
There's no magic stash of documentation that you've missed. What is on ReadTheDocs is what we have. Yes, there's a lot more documentation needed. We know. We're working on it.

**Comment by mhsmith** [comment]
In what way are they not sufficient? There's a "Reference" link on the [front page](https://toga.readthedocs.io/en/stable/index.html) which leads to some fairly complete API documentation.

**Comment by Gergely54** [comment]
Thanks Malcolm, I overlooked this topic.   
  

  
Kind regards
  
Gergely
  
  

  

      
  
  
>

**Comment by Gergely54** [comment]
Thanks for your reply. I was noticed on this...
  

  
https://toga.readthedocs.io/en/stable/index.html
  
  
Groet,
  
Gergely
  
Mob: +31 6 29007271
  

  

        
  
  
>
